contrail_api_cli_extra/misc/vm.py

```diff
@@ -47,7 +48,11 @@ def __call__(self, src=None, dst=None, **kwargs):
         vms = []
         src.fetch()
         for vm in src.refs.virtual_machine:
-            vm.fetch()
+            try:
+                vm.fetch()
+            except ResourceNotFound:
+                printo("virtual-machine/%s not found" % vm["uuid"])
```

Comment:
You can just print the exception message, it will be the same

except ResourceNotFound as err:
  printo(err)